src/scripts/components/wrapped-select.js

```diff
@@ -7,7 +7,10 @@ export default class WrappedSelect extends WrappedElement {
   }
 
   get placeholderOption() {
-    return this.element.querySelector('option[placeholder]');
+    return (
+      this.element.querySelector('option[value=""]') ||
+      this.element.querySelector('option[placeholder]')
+    );
```

Comment:
Would you mind just adding a comment here explaining why both are supported please 👍

Reply:
@jshjohnson done.

Reply:
Would you also mind incrementing a minor version please (not major - ignore my original comment )

Reply:
done (as a separate commit, as updating the build file for the new version also included other changes which were not yet in it).

Reply:
That should be done in the same `.querySelector('option[value=""], option[placeholder]')`

Reply:
@tinovyatkin no, that's not the same. Your proposal would use whatever is found first. My code is using the standard way if it is there (and so if you define both, the standard way wins)

Reply:
@jshjohnson 
Ugh, if somebody defines both, one with invalid, but very implicit and documented (by this library) attribute `placeholder` and second one just with empty value and empty value starts winning - I think it's a breaking change.